src/server.c

```diff
@@ -4099,13 +4112,7 @@ int processCommand(client *c) {
     }
 
     /* Exec the command */
-    if (c->flags & CLIENT_MULTI &&
-        c->cmd->proc != execCommand &&
-        c->cmd->proc != discardCommand &&
-        c->cmd->proc != multiCommand &&
-        c->cmd->proc != watchCommand &&
-        c->cmd->proc != quitCommand &&
-        c->cmd->proc != resetCommand)
+    if (c->flags & CLIENT_MULTI && isMultiQueuedCommand(c))
```

Comment:
now that the `if` is one line, let's pull the `{` to that same line.

Reply:
done